peachjam_api/serializers.py

```diff
@@ -95,11 +96,15 @@ class Meta:
             "repealed",
             "year",
             "taxonomies",
+            "languages",
         )
 
     def get_taxonomies(self, instance):
         return [x.topic.name for x in instance.taxonomies.all()]
 
+    def get_languages(self, instance):
+        return Work.objects.get(frbr_uri=instance.work_frbr_uri).languages
```

Comment:
Use `instance.work` otherwise you can't take advantage of prefetching the work when loading the documents.

Reply:
I've made the change